crates/matrix-sdk-crypto/src/machine.rs

```diff
@@ -4947,4 +5066,349 @@ pub(crate) mod tests {
             .unwrap();
         assert_matches!(thread_encryption_result, UnsignedDecryptionResult::Decrypted(_));
     }
+
+    #[async_test]
+    async fn test_decryption_trust_requirement() {
+        let (alice, bob) =
```

Comment:
This is now one of those unfocused tests we mentioned yesterday, especially since this whole setup section is repeated in the test bellow.

Could you turn this into a helper that gets called by both tests, there are other common places you could turn into helpers. That would also make it easier to make smaller, more focused tests.

Reply:
Added helpers.  It seems like a bunch of tests could be changed to use the `encrypt_message` helper, but I haven't searched to see which ones.  I can do that in this PR if you want.